--- joplin-2.3.5/packages/app-clipper/popup/package.json	2021-08-19 00:03:14.393138478 +0800
+++ joplin-2.3.5/packages/app-clipper/popup/package.json_back	2021-08-19 00:03:02.729026789 +0800
@@ -19,7 +19,7 @@
     "dotenv": "8.2.0",
     "dotenv-expand": "5.1.0",
     "file-loader": "4.3.0",
-    "fs-extra": "^8.1.0",
+    "fs-extra": "^6.0.1",
     "html-webpack-plugin": "4.0.0-beta.11",
     "identity-obj-proxy": "3.0.0",
     "jest": "24.9.0",
